src/services/formatting/formatting.ts

```diff
@@ -78,7 +78,7 @@ namespace ts.formatting {
         //  1. the end of the previous line
         //  2. the last non-whitespace character in the current line
         let endOfFormatSpan = getEndLinePosition(line, sourceFile);

```

Comment:
The following code looks like it now disagrees with the comment: without `!isLineBreak`, "1. the end of the previous line" will no longer happen.